ipn/ipnlocal/local.go

```diff
@@ -1359,10 +1360,6 @@ func (b *LocalBackend) SetControlClientStatus(c controlclient.Client, st control
 			prefsChanged = true
 		}
 	}
-	if st.URL != "" {
```

Comment:
Rebased and resolved a conflict with #13258 where I previously moved this block.